### ISSUE TITLE
Could not write header for output file #0 incorrect codec parameters

### ISSUE THREAD

**Issue by KuraudoXS** [YaoFANGUK/video-subtitle-remover#8]
Please find attached.

The video is not being created after subtitle removal is complete.
![Screenshot 11_19_2023 11_45_44 PM](https://github.com/YaoFANGUK/video-subtitle-remover/assets/9687549/b35f32d9-5950-46f5-92ef-067efdf42f58)
![Administrator_ C__vsr_启动程序 exe 11_19_2023 11_45_54 PM](https://github.com/YaoFANGUK/video-subtitle-remover/assets/9687549/3bc86ac2-ab5b-4d18-975f-494931f636a3)

**Comment by KuraudoXS** [comment]
I found the video in TEMP folder. However the audio is not attached. And the codec is different from the source file.

Source file codec:
![JENOVA_E MOV 0 avi - VLC media player 11_20_2023 2_58_21 PM](https://github.com/YaoFANGUK/video-subtitle-remover/assets/9687549/5791bf2e-2ed9-4be6-b080-f372514fef5b)
Output file codec: 
![Current Media Information 11_20_2023 3_06_15 PM](https://github.com/YaoFANGUK/video-subtitle-remover/assets/9687549/cb6caa7e-9c0d-4b02-a64a-da2530d64c51)

**Comment by YaoFANGUK** [comment]
thannk you for your input, this bug has been fixed. plz check

**Comment by KuraudoXS** [comment]
Thank you. Will that make it as new update 1.0.1? 

Also, see attached. One more question I've got: 
![vlcsnap-2023-11-22-22h00m31s343](https://github.com/YaoFANGUK/video-subtitle-remover/assets/9687549/e35c6b6e-aebe-4876-87a4-69e26fcc29bf)
After the first removal, one letter in three different frames sadly were not completely removed. Gently, could there be a function for this kind of situation?

**Comment by KuraudoXS** [comment]
fail to extract audio
[Finished]Subtitle successfully removed, video generated at：C:/Users/nomid/OneDrive/Desktop\JENOVA_E.MOV[0]_no_sub.mp4
time cost: 317.24s

The audio is failed sadly.
Also the video codec is different from the source codec. Please see attached.
![Current Media Information 11_23_2023 7_57_03 PM](https://github.com/YaoFANGUK/video-subtitle-remover/assets/9687549/ad4cedc0-41f5-434b-a768-85cc13fb3abf)

**Comment by YaoFANGUK** [comment]
cloud you send the video to me

**Comment by KuraudoXS** [comment]
Yes. See link for the video here: http://yatoshicom.free.fr/JENOVA_E.MOV[0].avi

**Comment by YaoFANGUK** [comment]
> Yes. See link for the video here: [http://yatoshicom.free.fr/JENOVA_E.MOV[0].avi](http://yatoshicom.free.fr/JENOVA_E.MOV%5B0%5D.avi)

sorry for the late reply, it's 404

**Comment by jinwyp** [comment]
I got same error :
使用源码部署的环境 运行后得到没有声音的视频文件。 报错如下


Your Paddle Fluid is installed successfully! Let's start deep Learning with Paddle Fluid now
1920 1080
Subtitle Removing: 100%|█████████▉| 1686/1687 [04:16<00:00, 19.04frame/s][adts @ 00000215d3aef4c0] Only AAC streams can be muxed by the ADTS muxer
Could not write header for output file #0 (incorrect codec parameters ?): Invalid argument
Error initializing output stream 0:0 --

Subtitle Removing: 100%|██████████| 1687/1687 [04:18<00:00,  6.54frame/s]
